pennylane/templates/subroutines/qpe.py

```diff
@@ -147,6 +147,10 @@ def _flatten(self):
         metadata = (self.hyperparameters["estimation_wires"],)
         return data, metadata
 
+    @classmethod
+    def _primitive_bind_call(cls, *args, **kwargs):
+        return cls._primitive.bind(*args, **kwargs)
```

Comment:
Even here I would say estimation wires could also be considered part of the target wires of the operation 🤔